### ISSUE TITLE
Getting error "enimEvt is not a function" when I close the modal by swiping down in react-native 0.63.2.

### ISSUE THREAD

**Issue by kishanbharda** [maxs15/react-native-modalbox#332]
I am using this component in my project. My react-native version is 0.63.2. Here the code that I have used to display modal : 
```js
// First I imported modal box 
import ModalBox from 'react-native-modalbox';

// created a ref to hold the modal
const changePasswordModal = useRef();

// Opening modal by clicking on one of my list item 
<ListItem
  accessoryLeft={(liArProps) => (
    <Icon {...liArProps} name="lock-outline" />
  )}
  accessoryRight={(liArProps) => (
    <Icon {...liArProps} name="chevron-right-outline" />
  )}
  title="Change Password"
  onPress={() => changePasswordModal.current.open()}
/>

// code that I used to define modal
<ModalBox
  ref={changePasswordModal}
  backButtonClose={true}
  style={styles.changePasswordModal}>
  <ChangePassword />
</ModalBox>

// modal style
changePasswordModal: {
  height: Dimensions.get('window').height / 2,
  width: Dimensions.get('window').width - 50,
},
```

But now, when I open a modal and close it by swiping down it crash the app and displaying me an error : 
![image](https://user-images.githubusercontent.com/39869106/95162369-283c7500-07c3-11eb-859a-755e770781d7.png)

This is only happing if I close modal by swiping down, If I close modal by clicking on backdrop it's closing properly without any error.

**Comment by R1D3** [comment]
same issue for me, it only happens since this morning ....

**Comment by winardis** [comment]
i just updated to react native 0.63.3 and having the same issue as you are

**Comment by januarsmad** [comment]
set useNativeDriver to false. it works on me..

**Comment by iabduul7** [comment]
> set useNativeDriver to false. it works on me..

where to put this line?

**Comment by uzaircs** [comment]
Same issue

`TypeError: animEvt is not a function`

Setting  `useNativeDriver={false}` does work but causes opening animation to disappear. The modal just appears without any animation after a slight delay.

I have set `swipeToClose={false}` for now until I see a better solution for this.

**Comment by uzaircs** [comment]
> > set useNativeDriver to false. it works on me..
> 
> where to put this line?

In the Modal Component

```
   <Modal
          useNativeDriver={false}
          ref={"modal1"}
          //other props
    >
   </Modal>
```

**Comment by iabduul7** [comment]
> > > set useNativeDriver to false. it works on me..
> > 
> > 
> > where to put this line?
> 
> In the Modal Component
> 
> ```
>    <Modal
>           useNativeDriver={false}
>           ref={"modal1"}
>           //other props
>     >
>    </Modal>
> ```
> 
> [Or see this answer](https://github.com/maxs15/react-native-modalbox/issues/332#issuecomment-706104379)

Okay, got it

**Comment by maxs15** [comment]
Fixed in version 2.0.2 👍
